mobile/voice_journal_app/lib/stats.dart

```diff
@@ -0,0 +1,253 @@
+import 'package:flutter/material.dart';
+import 'package:fl_chart/fl_chart.dart';
+import 'theme.dart'; // Make sure this is correctly pointing to your theme.dart
+
+enum TimeFrame { all, year, month, week }
+enum Emotions {
+ 	happiness, // id = 0
+  sadness, // id = 1
+  anger, // id = 2
+  fear, // id = 3
+  disgust, // id = 4
+  surprise, // id = 5
+}
```

Comment:
Let's reuse the [Emotion enum that Alec created](https://github.com/LaurierCS/POD1/blob/a7abd79734e5fa335a869e811328b8a7b8ae23ae/mobile/voice_journal_app/lib/Emotions_enums.dart). Since his PR hasn't gone through yet simply create the file again exactly the same and there shouldn't be many merge conflicts. You can then import it into this file.